en/starter/generator/project_layout_reference.md

```diff
@@ -150,6 +161,14 @@ files will be included in the scaffolded project:
 ├── Dockerfile                      - Docker spec for an image to run the application
 ├── Dockerfile-tools                - Docker spec for an image to build the application
 ├── .dockerignore                   - Defined files to ignore for building images
+├── chart/
```

Comment:
You've added the chart directory in here but there is no reference to it or mention of it here http://localhost:4000/en/starter/generator/core_concepts.html#docker-capability which there should be as we talk about the other files and what they are for, so chart needs to be added including an explanation of what files it contains and what they are for.